mlctl/jobs/MlctlDeployJob.py

```diff
@@ -13,7 +12,7 @@ def __init__(self, job_type, project, name=None):
             self.name = name
         else:
             # else make a new name randomly
-            word = RandomWords().random_word('d')
+            word = RandomWords().random_word()
```

Comment:
Random word('d') was intentional because it picked words that start with d

Reply:
Understood

Not a big fan of using words, if we look at the implementation it loads the random words from a file and it is just a fixed list of words

https://github.com/tomislater/RandomWords/blob/master/random_words/nouns.dat

And the list of words are bigger or smaller depending on which character we choose

Generally it is better to stick with random generator based on epoch, which is O(1) in time and memory and no extra disk access, and has much lower chance of collision, which I believe is the reason for going random in the first place

I will suggest we move to something cheaper and have lower chance of collision, and avoid potential bug for customer

Reply:
I agree with Clarence. Let's revisit before merging it to main.